book/src/usage.md

```diff
@@ -2,7 +2,7 @@
 
 (Currently not fully documented, see the [keymappings](./keymap.md) list for more.)
 
-See [tutor](https://github.com/helix-editor/helix/blob/master/runtime/tutor) (accessible via `hx --tutor` or `:tutor`) for a vimtutor-like introduction.
+See [tutor](https://github.com/helix-editor/helix/blob/master/runtime/tutor) (accessible via `hx --tutor` or `:tutor`) for a Vimtutor-like introduction.
```

Comment:
vimtutor could either be interpreted as the name of the command (`vimtutor`) or as the proper noun ("Vim tutor"). I would prefer just surrounding the lowercase version in backticks since it's a command - that will work better with the hyphen.